src/sage/plot/misc.py

```diff
@@ -135,9 +135,15 @@ def setup_for_eval_on_grid(funcs,
         vars = [r[0] for r in ranges]
         ranges = [r[1:] for r in ranges]
         if len(set(vars)) < len(vars):
-            raise ValueError("range variables should be distinct, but there are duplicates")
+            raise ValueError("range variables should be distinct, but there are duplicates")   
```

Comment:
> It seems a new PR was suggested here by the same author: #35112 (now closed) and #35113 Creating new PRs like this split the current discussion across many webpages and makes the discussion harder to follow. Can't we just continue the discussion here?
> 
> Isn't it possible to force push a branch here instead of pushing it on a new PR?

It might very well be possible.

I was having trouble making desired aesthetic changes and then squashing the old commits so thought it might be better to create new PRs with cleaner commit histories.

I learn up on GitHub actions to avoid such cases in the future.